src/mixins/filing-template-mixin.ts

```diff
@@ -254,6 +254,10 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
 
     // restore business name data
     const nameRequest = draftFiling.amalgamationApplication.nameRequest as NameRequestFilingIF
+
+    if (this.getAmalgamationType !== AmalgamationTypes.REGULAR) {
+      this.setCorrectNameOption(CorrectNameOptions.CORRECT_AML_ADOPT)
+    }
```

Comment:
OK, I think we can leave this as a fallback option (say, for existing amalgamation drafts). This should still be pre-populated in Filings UI [around here](https://github.com/bcgov/business-filings-ui/blob/1142374aa156d23d96305144b85ddbdedf50d1c0/src/views/AmalgamationSelection.vue#L239).

In fact, check with Yui whether he wants the "adopt" option pre-populated by default for regular amalgamations that are started from an existing business (as above).

One last thing: on line 258, instead of assuming it must be a short-form alamg if it's not a regular, please check if it's a short-form amalg. This kind of "if not something" has bitten us in the past when we added a type or filing or something and the assumption is wrong. It's better to check "if something".

Reply:
Checked with Yui: "We don't want to assume adoption of name for regular when started from an existing business."

Reply:
Thanks for checking with Yui. So this code applies only to horiz and vert amalgamations 👍